tap_linkedin_ads/discover.py

```diff
@@ -18,3 +18,13 @@ def discover():
         ))
 
     return catalog
+
+def check_accounts_list(config):
+    if config.get('accounts'):
+        account_list = config['accounts'].replace(" ", "").split(",")
+        for account in account_list:
+            try:
+                int(account)
+            except ValueError as e:
+                message = "The accounts provided in the configuration are not valid numbers."
```

Comment:
@savan-chovatiya Improvise the message that "The account '{}' provided in the configuration are having non-numeric values.".format(account)"

Reply:
Changed message as per suggestion.